language/stdlib/modules/LibraSystem.move

```diff
@@ -90,12 +90,28 @@ module LibraSystem {
     // This copies the vector of validators into the LibraConfig's resource
     // under ValidatorSet address
     fun set_validator_set(value: LibraSystem) acquires CapabilityHolder {
+        assert(
+            exists<CapabilityHolder>(CoreAddresses::LIBRA_ROOT_ADDRESS()),
```

Comment:
Is this function called from genesis (initialize)? If not you can just add `LibratimeStamp::assert_operating()` and the according schema to the spec. By the invariant, it follows then that the capability exists, and you don't need this assert.